packages/manager/modules/dedicated-universe-components/src/config/index.js

```diff
@@ -0,0 +1,16 @@
+import { Environment } from '@ovh-ux/manager-config';
+import constantsConfig from './constants.config';
+
+const aapiRootPath = '/engine/2api/';
+const swsProxyRootPath = 'apiv6/';
+
+const target = Environment.getRegion();
+// const prodMode = !!WEBPACK_ENV.production;
```

Comment:
```suggestion

```
this looks like it should be kept in `dedicated` app

Reply:
yes its added to the dedicated app